cmd/prysmctl/withdrawal/withdrawal.go

```diff
@@ -0,0 +1,165 @@
+		"of changing your bls withdrawal address to an ethereum address. " +
+		"THIS ACTION WILL NOT BE REVERSIBLE ONCE INCLUDED. " +
+		"You will NOT be able to change the address again once changed. ")
+	for _, foundFilePath := range foundFilePaths {
+		b, err := os.ReadFile(filepath.Clean(foundFilePath))
+		if err != nil {
+			return errors.Wrap(err, "failed to open file")
+		}
+		switch string(b)[0:1] {
+		case "[":
+			var to []*apimiddleware.SignedBLSToExecutionChangeJson
+			if err := json.Unmarshal(b, &to); err != nil {
+				return errors.Wrap(err, "failed to unmarshal file")
+			}
+			if len(to) == 0 {
+				return errors.New("the list of signed requests is empty")
+			}
+			for _, jsonOb := range to {
+				if err := verifyWithdrawalCertainty(r, jsonOb); err != nil {
+					return err
+				}
+			}
+			return callWithdrawalEndpoint(ctx, BeaconNodeHost, to)
+		case "{":
+			var to *apimiddleware.SignedBLSToExecutionChangeJson
+			if err := json.Unmarshal(b, &to); err != nil {
+				return errors.Wrap(err, "failed to unmarshal file")
+			}
+			if to == nil || to.Message == nil {
+				return errors.New("the object or object's message field in file is empty")
+			}
+			if err := verifyWithdrawalCertainty(r, to); err != nil {
+				return err
+			}
+			return callWithdrawalEndpoint(ctx, BeaconNodeHost, []*apimiddleware.SignedBLSToExecutionChangeJson{to})
+		default:
+			return errors.New("the provided file is not a json object or list of jason objects")
+		}
+	}
+	return nil
+}
+
+func verifyWithdrawalCertainty(r io.Reader, request *apimiddleware.SignedBLSToExecutionChangeJson) error {
+	au := aurora.NewAurora(true)
+	withdrawalConfirmation := request.Message.ToExecutionAddress
+	fmt.Println(au.Red("===================================="))
+	fmt.Println("YOU ARE ATTEMPTING TO CHANGE THE BLS WITHDRAWAL(" + au.Red(request.Message.FromBLSPubkey).String() + ") ADDRESS " +
+		"TO AN ETHEREUM ADDRESS(" + au.Red(request.Message.ToExecutionAddress).String() + ") FOR VALIDATOR INDEX(" + au.Red(request.Message.ValidatorIndex).String() + "). ")
+	_, err := withdrawalPrompt(withdrawalConfirmation, r)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func callWithdrawalEndpoint(ctx context.Context, host string, request []*apimiddleware.SignedBLSToExecutionChangeJson) error {
+	body, err := json.Marshal(request)
+	if err != nil {
+		return errors.Wrap(err, "failed to marshal json")
+	}
+	fullpath := host + basePath + apiPath
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, fullpath, bytes.NewBuffer(body))
+	if err != nil {
+		return errors.Wrap(err, "invalid format, failed to create new Post Request Object")
+	}
+	req.Header.Set("Content-Type", "application/json")
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("API request to %s , responded with a status other than OK, status: %v body: %v", fullpath, resp.Status, resp.Body)
```

Comment:
```suggestion
		return fmt.Errorf("API request to %s responded with a status other than OK - status %v, body %v", fullpath, resp.Status, resp.Body)
```
Error messages should not contain colons because that's how errors are separated.